docker_files/etc/confd/templates/config.json.tmpl

```diff
@@ -0,0 +1,142 @@
+{
+	"wsPort": {{getv "/lisk/wsport" "8001"}},
+	"httpPort": {{getv "/lisk/httpport" "8000"}},
+	"address": "{{getv "/lisk/address" "0.0.0.0"}}",
+	"version": "{{getenv "LISK_VERSION"}}",
+	"minVersion": "{{getenv "LISK_MIN_VERSION"}}",
```

Comment:
If these **env variables** meant to run container with different min values, then I believe its not the right case. Once the docker container is built, its built for a specific version, which is bound to a specific minVersion. User should not be allowed to change those on runtime.

Reply:
> Once the docker container is built, its built for a specific version, which is bound to a specific minVersion.
That is exactly what will happen, those environment variables will come from the docker image (see https://github.com/LiskHQ/lisk/blob/83de84142be1c7603941278be0d51f0a6c833abb/Dockerfile#L44)